create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -382,14 +385,43 @@ void shouldEmitMessageToEventReferenceContainingS3UriPointingToNewCreatePublicat
         assertThat(request, is(equalTo(expectedRequest)));
     }
 
+    @Test
+    void shouldExtractJournalArticle() throws IOException {
+        scopusData = scopusData.create(CitationtypeAtt.AR);
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusData.toXml());
+        var s3Event = createS3Event(uri);
+        CreatePublicationRequest createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
+        var actualPublicationInstance =
+            createPublicationRequest.getEntityDescription().getReference().getPublicationInstance();
+        assertThat(actualPublicationInstance, isA(JournalArticle.class));
+    }
+
+    @ParameterizedTest
```

Comment:
@ParameterizedTest(name ="should not generate CreatePublicationRequest when CitationType is not:{0}")

Reply:
fixed